integr8ly/installation
# INTLY-2190 Apply ksm alerts to ensure we have all the latest rules after upgrade

## Additional Information
Follow up from https://github.com/integr8ly/installation/pull/724 (upgrade for changes in https://issues.jboss.org/browse/INTLY-2190)

This PR changes the way the upgrade of the ksm-alerts PrometheusRule is done.

*Before*

The CR was exported, modified using sed, then reimported.

*After*

The CR is applied to the cluster (`oc apply`) to ensure it has the latest changes.

*Rationale*

This way of upgrading of individual rule changes is safe and less error prone.
The CR should not be modified in the cluster outside of the install or upgrade. We can't allow for any changes outside of the install & upgrade.
Therefore, the CR can simply be updated in-situ.
The prometheus-operator will reconcile the latest alerts and get any changes we've made.


## Verification Steps

1. Install from tag `release-1.4.1`
2. Checkout this branch
2. Run the upgrade playbook
3. Verify the alert changes in https://github.com/integr8ly/installation/pull/707 & https://github.com/integr8ly/installation/pull/724 have been applied and are showing up correctly in Prometheus UI > Alerts tab. This may take a few minutes to update
